src/test/parse-fail/obsolete-proc.rs

```diff
@@ -10,8 +10,8 @@
 
 // Test that we generate obsolete syntax errors around usages of `proc`.
 
-fn foo(p: proc()) { } //~ ERROR obsolete syntax: the `proc` type
+fn foo(p: proc()) { } //~ ERROR: the `proc` type isn't used for the moment
```

Comment:
Ah no, I always stop at stage1. And even if this error message isn't generated, a nice comment is always useful. ^^ Or it wasn't what you meant ?